Gruntfile.js

```diff
@@ -70,9 +70,16 @@ module.exports = function(grunt) {
             }
         },
 
-        jsFiles: [
+        jsHeadFiles: [
             // Include:
+            'source/js/head-scripts.js'
+        ],
+
+        jsFootFiles: [
+            // Include:
+            'source/js/jquery-2.0.0b2.js',
```

Comment:
I guess my thinking is that the CMS is probably already pulling this somewhere else. If we add it here, it will at best add to the page weight, and at worst cause some kind of conflicts.